### PR TITLE
Fix timeout cast exception

### LINKED ISSUE #684: Property timeout causes a class cast exception if is set
If the timeout property is available to override the 30s for waiting for the deploy to be done, i tracked down to the task DeployTask  on line https://github.com/OpenLiberty/ci.gradle/blob/0b72dc4f92df09fbb9896f35d230bb503db9621c/src/main/groovy/io/openliberty/tools/gradle/tasks/DeployTask.groovy#L646, the property is defined as String but it's used as long, should this property used on liberty start as well why exist two properties to do the same thing timeout and verifyAppStartTimeout

### PR DESCRIPTION
**Pull request by mattbsox**
Fixes #684

### CODE REVIEW

**Review by cherylking — COMMENTED** [review]
Should we check that it is a long before calling `longValue()`? We have code in DevTask that checks `isInteger()` before calling `Long.valueOf()` on the `server.timeout`.

**Comment by mattbsox** [comment]
Yeah, I'll add a check for that.

File: src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy [hunk 1]
```diff
@@ -925,7 +925,12 @@ class DevTask extends AbstractFeatureTask {
 
         if (serverStartTimeout == null) {
             if (server.timeout != null && server.timeout.isInteger()) {
-                serverStartTimeout = server.timeout.toInteger();
+                try {
```
Comment (cherylking) [inline src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy:928]: I don't believe `isInteger()` is valid on the line before this either. Probably should just check that the string is notEmpty().

**Review by cherylking — CHANGES_REQUESTED** [review]

**Review by cherylking — APPROVED** [review]

File: src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy (hunk 1 above)
Comment (mattbsox) [reply src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy:928]: Yeah, missed that. Fixed it.

**Pull request merged**